admin/interface-yoast-notification-center.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
There is no real use for this interface. I would remove it. There is no contract to be enforced.